android/cli/commands/_build.js

```diff
@@ -2817,91 +2819,51 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 		if (!jsFilesToEncrypt.length) {
 			// nothing to encrypt, continue
 			return next();
-		}
-
-		// figure out which titanium prep to run
-		let titaniumPrep = 'titanium_prep';
-		if (process.platform === 'darwin') {
-			titaniumPrep += '.macos';
-		} else if (process.platform === 'win32') {
-			titaniumPrep += '.win32.exe';
-		} else if (process.platform === 'linux') {
-			titaniumPrep += '.linux' + (process.arch === 'x64' ? '64' : '32');
-		}
+		} else if (cloak) {
```

Comment:
Minor style nitpick I prefer:
The else is not necessary because the block above returns. Then you could invert the if check to handle the error case and return/exit early, i.e.
```javascript
if (!cloak) {
  this.logger.error('Could not encrypt assets!');
  process.exit(1);
}

// the main logic for encrypting goes here now...
```

Personally I prefer to handle early exists/pre-condition checks at top and avoid nesting the main codepath/logic as a result.